src/test/java/fi/otavanopisto/kuntaapi/test/AbstractIntegrationTest.java

```diff
@@ -89,7 +84,7 @@ protected void waitApiListCount(String path, int count) throws InterruptedExcept
         }
         
         if ((counter % 10) == 0) {
-          logger.log(Level.WARNING, () -> String.format("... still waiting %d items in %s, current count %d", count, path, listCount));
+          System.out.println(String.format("... still waiting %d items in %s, current count %d", count, path, listCount));
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS106)